flake.lock

```diff
@@ -0,0 +1,39 @@
+{
+  "nodes": {
+    "bitcoin-transactions-src": {
```

Comment:
The narhash is out of date for me. It's capitalizing on annoying bug with `path` where the narhash needs to be updated every  time the repo changes. You'd need to run `nix flake lock --update-input bitcoin-transactions-src` to correct.